www/_posts/2016-01-27-tools-release.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Cordova 6.0.0 Released!"
+categories: news
+tags: release tools
+---
+New versions of cordova tools are now live!
+
+* [cordova-lib@6.0.0](https://www.npmjs.org/package/cordova-lib)
+* [cordova@6.0.0](https://www.npmjs.org/package/cordova)
+* [plugman@1.1.0](https://www.npmjs.org/package/plugman)
+* [cordova-js@4.1.3](https://www.npmjs.org/package/cordova-js)
+
+Release Highlights include:
+* Updated default platform versions to `cordova-android@5`, `cordova-ios@4` and `cordova-windows@4.3`. 
+    - `cordova-android@5` supports **Android marshmallow** 
+    - `cordova-ios@4` adds **iOS9** and `WKWebView` support
+    - `cordova-windows@4` updated the platform to use the new Platform API.
```

Comment:
While technically, not incorrect - I would recommend providing specific major versions here too especially for Windows & iOS - 4.3 and 5.1